### PR TITLE
John conroy/fair data messaging HMP-202

### PR DESCRIPTION
**Pull request by john-conroy**
Add messaging for fair data principles to homepage, Vitessce, and the dataset file browser.

Homepage Data Use Guidelines

Screen Shot 2023-06-30 at 3 28 23 PM

File Browser

Screen Shot 2023-06-30 at 3 29 25 PM

Vitessce

Screen Shot 2023-06-30 at 3 29 13 PM

Screen Shot 2023-06-30 at 3 29 06 PM

### CODE REVIEW

**Comment by john-conroy** [comment]
I had to fix a cypress test that was getting a false positive on the presence of 'log in' from some text.

File: end-to-end/cypress/e2e/portal/portal-ui.cy.js [hunk 1]
```diff
@@ -28,9 +28,11 @@ describe('portal-ui', () => {
 
       // Static pages are tested separately.
 
+      cy.findByTestId('user-profile-dropdown').contains('User Profile').click();
+
       // login
-      cy.contains('login');
       // Don't click! We shouldn't depend on Globus in tests.
+      cy.findByTestId('login-link').contains('Log In').click();
```
Comment (NickAkhmetov) [inline end-to-end/cypress/e2e/portal/portal-ui.cy.js:35]: Does this test pass without the `click`?

File: context/app/static/js/shared-styles/sections/SectionFooter/SectionFooter.stories.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+export default {
+  title: 'Sections/SectionFooter',
+  component: SectionFooter,
+};
+
+export const Basic = {
+  args: {
+    items: [
+      { key: '1', component: <>Item 1</> },
+      { key: 2, component: <>Item 2</> },
```
Comment (NickAkhmetov) [inline context/app/static/js/shared-styles/sections/SectionFooter/SectionFooter.stories.js:14]: Not a big deal, but may be a good idea to have consistency between the `key` prop types

**Review by NickAkhmetov — DISMISSED** [review]
Looks good overall, some minor comments:

File: end-to-end/cypress/e2e/portal/portal-ui.cy.js (hunk 1 above)
Comment (john-conroy) [reply end-to-end/cypress/e2e/portal/portal-ui.cy.js:35]: Whoops, that was an unintentional result of copy and paste. I'll remove the click here.

File: context/app/static/js/shared-styles/sections/SectionFooter/SectionFooter.stories.js (hunk 2 above)
Comment (john-conroy) [reply context/app/static/js/shared-styles/sections/SectionFooter/SectionFooter.stories.js:14]: Thanks!

**Review by NickAkhmetov — APPROVED** [review]
Thanks for making those adjustments! Should be all set to 🚀!

**Review by lchoy — APPROVED** [review]

**Review by NickAkhmetov — APPROVED** [review]
Re-approving post-merge

**Pull request merged**